### PR TITLE
CHANGELOG: Document changes since v0.1.0

### PR DESCRIPTION
**Pull request by wking**
Through f25e53a4 (Merge pull request #388, 2018-10-12).  Once this lands, we can tag a v0.2.0.

CC @abhinavdahiya, @crawford.

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
#374 might be required for v0.2.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,71 @@ All notable changes to this project will be documented in this file.
...
+- Asset state is preserved between invocations, allowing for a staged
+    install like:
+
+    ```console
+    $ openshift-install --dir=example manifests
+    $ $EDITOR emacs example/manifests/cluster-config.yaml
+    $ openshift-install --dir=example install-config
+    ```
+
+    which created a cluster wit ha customized `cluster-config.yaml`.
```
Comment (chancez) [inline CHANGELOG.md]: `s/wit ha/with a/`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,71 @@ All notable changes to this project will be documented in this file.
...
+- [The kube-apiserver][kube-apiserver-operator] and
+  [kube-controller-manager][kube-controller-manager-operator]
+  operators are called to render additional cluster manifests.
+- etcd is now avialable as a service in the `kube-system` namespace,
```
Comment (chancez) [inline CHANGELOG.md]: s/avialable/available/

File: CHANGELOG.md (hunk 1 above)
Comment (wking) [reply CHANGELOG.md]: > `s/wit ha/with a/`

Fixed in 625367d -> ec34840.

File: CHANGELOG.md (hunk 2 above)
Comment (wking) [reply CHANGELOG.md]: > s/avialable/available/

Fixed in 625367d -> ec34840.

**Comment by wking** [comment]
> #374 might be required for v0.2.0

Do we expect it to land today?  I'm fine cutting v0.3.0 next week or whenever it does land?

**Comment by abhinavdahiya** [comment]
> Do we expect it to land today? I'm fine cutting v0.3.0 next week or whenever it does land?

Sure if we are trying to do weekly. lets go ahead then.
/lgtm

**Pull request merged**